lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -427,39 +427,50 @@ defmodule ExUnit.DocTest do
   ## Extraction of the tests
 
   defp extract(module) do
-    all_docs = Code.get_docs(module, :all)
+    case Code.fetch_docs(module) do
+      {:docs_v1, line, _, _, moduledoc, _, docs} ->
```

Comment:
Is `line` always going to be a line number? The spec of `Code.get_docs` defines it as `:erl_anno.anno()` which might have more data in it. Should we use `:erl_anno.line(anno)` to get the line here?

Reply:
That would certainly be safer. However, in doc tests we know that it's just a line since it was generated by the Elixir compiler where we don't include more information. But since that could change I think it's best to go with your advice.

Reply:
Yeah, lets go with `:erl_anno.line/1`.